jupyter-web-app/rockcraft.yaml

```diff
@@ -1,88 +1,110 @@
+# Based on https://github.com/kubeflow/kubeflow/blob/master/components/crud-web-apps/jupyter/Dockerfile
 name: jupyter-web-app
 summary: An image for Jupyter UI
 description: |
   This image is used as part of Charmed Kubeflow product. Jupyter UI web application provides
   users with web UI to access and manipulate Jupyter Notebooks in Charmed Kubeflow.
-version: v1.7.0_22.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
+version: v1.7.0_20.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
```

Comment:
if you're keeping this image internally, then it might not be that relevant, but adding app-specific strings after <base-version> could conflict with our tagging scheme moving forward. For now, although unofficial, we're adopting the same tagging schema as snap channels, and thus the last digit is reserved for the image revision and is thus dynamic.

Reply:
Comment might be misleading. I will update it. That last digit is version of the ROCK itself, not the application, i.e. with every update to rockcraft.yaml we will increment it.

Reply:
ok. still, that's not the same as a revision, so please keep that in mind in case you ever want to publish those through OCI factory and the official channels

Reply:
Understood. We will have to go with the current versioning scheme according to our KF versioning spec. When we get to publishing to OCI factory, we would have to review it.